plugin-server/src/worker/vm/extensions/jobs.ts

```diff
@@ -48,15 +47,15 @@ export function createJobs(server: Hub, pluginConfig: PluginConfig): Jobs {
                 pluginConfigId: pluginConfig.id,
                 pluginConfigTeam: pluginConfig.team_id,
             }
-            await server.graphileWorker.enqueue(
-                JobName.PLUGIN_JOB,
-                job,
-                {
-                    key: 'plugin',
-                    tag: pluginConfig.plugin?.name ?? '?',
-                },
-                true
-            )
+            await server.kafkaProducer.queueMessage({
+                topic: 'jobs',
+                messages: [
+                    {
+                        key: pluginConfig.team_id.toString(),
```

Comment:
Didn't comment about this on the last PR but one thing we should probably do is get a sense of how well this ends up distributing the load across partitions

Reply:
We could go for the pluginConfig id. I suspect we don't need to ensure the ordering too strictly within a team. Perhaps we don't need to do so within a plugin either?